### ISSUE TITLE
Missing js and img assets

### ISSUE THREAD

**Issue by itcarroll** [datacarpentry/python-ecology-lesson#92]
Note, this applies to all of the *-ecology-lessons. I cannot tell where the origin of the shared files (_layouts, etc.) is though, so I'm raising the issue here for now.

If you look at the browser JS console on loading datacarpentry.github.io/python-ecology-lesson, there are three resources not found:
http://www.datacarpentry.org/python-ecology-lesson/js/jquery-1.9.1.min.js
http://www.datacarpentry.org/python-ecology-lesson/js/bootstrap/bootstrap.min.js
http://www.datacarpentry.org/img/main_shadow.png

The first two requests come from `_includes/javascript.html` and the last from `css/swc.css`. They are, indeed, missing from the repository.

If a maintainer can describe the usual workflow from updating these generic components from upstream, I'd be happy to follow up with a pull request.

**Comment by ctb** [comment]
when looking at #123 and using `bundle exec jekyll serve` to serve the site, I saw the following errors:

```
[2016-12-29 13:53:10] ERROR `/js/jquery-1.9.1.min.js' not found.
[2016-12-29 13:53:10] ERROR `/img/main_shadow.png' not found.
[2016-12-29 13:53:10] ERROR `/js/bootstrap/bootstrap.min.js' not found.
[2016-12-29 13:53:15] ERROR `/js/jquery-1.9.1.min.js' not found.
[2016-12-29 13:53:15] ERROR `/js/bootstrap/bootstrap.min.js' not found.
[2016-12-29 13:53:15] ERROR `/img/main_shadow.png' not found.
[2016-12-29 13:53:19] ERROR `/js/jquery-1.9.1.min.js' not found.
[2016-12-29 13:53:19] ERROR `/js/bootstrap/bootstrap.min.js' not found.
[2016-12-29 13:53:19] ERROR `/img/main_shadow.png' not found.
```

so that's another way to detect missing files ;).

**Comment by ctb** [comment]
I used `git grep` to find the location of the references --

```
% git grep jquery
_includes/javascript.html:<script src="{{page.root}}/js/jquery-1.9.1.min.js"></script>
% git grep bootstrap.min.js
_includes/javascript.html:<script src="{{page.root}}/js/bootstrap/bootstrap.min.js"></script>
% git grep main_shadow
css/swc.css:    background: url("/img/main_shadow.png") repeat-x scroll center top #FFFFFF;
```

It doesn't *seem* like JQuery is used anywhere on the site, and in any case if it's being used I'm guessing the site isn't obviously broken, so I am guessing that it can be removed cleanly. I'm not sure where `_layouts/bootcamp.html` and `_layouts/lesson.html` came from but that's where `javascript.html` is referenced:

```
% git grep javascript.html
_layouts/bootcamp.html:    {% include javascript.html %}
_layouts/lesson.html:    {% include javascript.html %}
```

Easy enough to submit a PR if people are ok with these being removed.

**Comment by wrightaprilm** [comment]
That sounds great, @ctb. Cleaning up missing assets and deprecated things has been on my to-do list, but I keep getting side-tracked with all the awesome updates our contributors  have been submitting :)

**Comment by ctb** [comment]
We do still need to find `main_shadow.png`, or otherwise figure out how to update the styles - I'm not sure we should just remove it (?).

**Comment by ctb** [comment]
See #125. I'd prefer to do @itcarroll's suggestion of updating from a central styles location tho - is that https://github.com/swcarpentry/styles?

**Comment by wrightaprilm** [comment]
I think it is, and I agree. I took a look at my Jan-Feb schedule and shorted out, so I'm vanishing for the weekend. If no one else gets around to it, I'll look into centralizing it when I'm back Monday.

**Comment by ctb** [comment]
On Fri, Dec 30, 2016 at 09:54:04AM -0800, April Wright wrote:
> I think it is, and I agree. I took a look at my Jan-Feb schedule and shorted out, so I'm vanishing for the weekend. If no one else gets around to it, I'll look into centralizing it when I'm back Monday.

heh, have a good weekend :)

**Comment by ctb** [comment]
See #126 for an initial attempt.

**Comment by qjcg** [comment]
This issue appears to be fixed following #130.
